site/templates/film/item.jet

```diff
@@ -71,7 +71,8 @@
                       <p>
                         {{range index, member := film.Cast}}
                           {{ path := "/search.html?q=" + member.Name }}
-                          <a href="{{ routeToPath(path) }}">{{member.Name}}</a>{{if index < len(film.Cast) - 1}}, {{end}}
+                          <a href="{{ routeToPath(path) }}">{{member.Name}}</a>{{if len(member.Character) > 0}} (as {{member.Character}}) {{end}}
```

Comment:
We can't assume that `as` is valid for every language.  You'll have to add a translation key for it or just remove it!

Reply:
Good point